src/ZoneServer/Scripting/AI/AiScript.cs

```diff
@@ -358,6 +375,14 @@ protected ICombatEntity GetMostHated()
 		/// <returns></returns>
 		protected bool IsHating(ICombatEntity entity)
 		{
+			// Always hating the person that casted this buff
+			if (this.Entity.IsBuffActive(BuffId.ProvocationImmunity_Debuff))
```

Comment:
I hope I don't need to mention that there are other cases still where you might want to use `TryGetBuff` for more concise code^^

Reply:
I also removed this one.